addon/components/flexberry-file.js

```diff
@@ -308,18 +308,18 @@ export default FlexberryBaseComponent.extend({
     Component's input additional CSS-class names.
     See [Semantic UI inputs classes](http://semantic-ui.com/elements/input.html).
 
-    @property class
+    @property inputClass
     @type String
-    @default 'fluid'
+    @default ''
     @example
```

Comment:
Remove this style:
".flexberry-lookup,
  .ui.input > input {
    width: 100%;
  }"